src/modules/enrollment/components/EnrollmentAssessmentActionButtons.tsx

```diff
@@ -92,9 +97,14 @@ const NewAssessmentMenu: React.FC<Props> = ({ enrollment }) => {
         : []),
       ...bottomItems,
     ];
-  }, [enrollment, getPath]);
+  }, [data, getPath]);
 
-  if (isEmpty(items)) return null;
+  if (items.length === 0)
+    return (
+      <LoadingButton loading={loading} disabled={!loading}>
```

Comment:
Nit: it would be nice if there was a tooltip explaining `Cannot to perform new assessments for exited enrollment.` if it is disabled. I we use the ButtonTooltipContainer for that